src/clr-angular/progress/spinner/spinner.ts

```diff
@@ -3,20 +3,27 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-import { Component, Input, HostBinding } from '@angular/core';
+import { Component, Input, HostBinding, ElementRef, AfterViewChecked } from '@angular/core';
 import { isBooleanAttributeSet } from '../../utils/component/is-boolean-attribute-set';
+import { AriaLiveService, AriaLivePoliteness } from '../../utils/a11y/aria-live.service';
 
 @Component({
   selector: 'clr-spinner',
+  providers: [AriaLiveService],
   template: `
     <ng-content></ng-content>
   `,
   host: {
-    '[attr.aria-live]': 'setAriaLive',
     '[attr.aria-busy]': 'true',
   },
 })
-export class ClrSpinner {
+export class ClrSpinner implements AfterViewChecked {
+  constructor(private el: ElementRef, private ariaLiveService: AriaLiveService) {}
+
+  ngAfterViewChecked() {
+    this.ariaLiveService.announce(this.el.nativeElement, this.ariaLive);
+  }
```

Comment:
The spinner has the same change detection issue as the progress bar but can simply be updated to use `ngAfterViewInit` instead of `ngAfterViewChecked`